src/main/resources/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig/config.jelly

```diff
@@ -17,5 +17,7 @@
         <f:entry title="${%Delete Stashes}" field="deleteStashes">
             <f:checkbox readonly="true"/>
         </f:entry>
+        <f:validateButton title="Validate S3 Bucket configuration" progress="Validate..." method="validateS3BucketConfig"
```

Comment:
Is this too expensive to just do automatically in a `doCheck*` method?

Reply:
it is expensive, it takes a few seconds.